index.js

```diff
@@ -63,7 +84,7 @@ app.post('/postToPublic',function(req, response){
 			    sender: 
 				{
 					name: 'Susi',
-					avatar: 'https://www.google.co.in/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwirq6TRoYvTAhWMLI8KHSgVCXsQjRwIBw&url=https%3A%2F%2Fjigyasagrover.wordpress.com%2F2016%2F05%2F26%2Floklak-brief-one-googlesummerofcode-fossasia%2F&bvm=bv.151426398,d.c2I&psig=AFQjCNG01WP05rdho79rkRnGvuwTzs8_hA&ust=1491411117179108' 
+					avatar: 'https://lh3.googleusercontent.com/-s1cRRcPxqsk/WTZg8CM828I/AAAAAAAAAvY/DglrI6rZkMktQEm_2RE637Isf7WGLp67QCLcB/s1600/susi.png' 
```

Comment:
What is this dependency on Google? Please don't create dependencies on other services. This is sharing our data with Google for no reason.